### ISSUE TITLE
Link topline summary template on Platform

### ISSUE THREAD

**Issue by claytonzook** [department-of-veterans-affairs/va.gov-team#42681]
## User Story
As a VFS researcher, I want all the important guidance I need in one location, that I know to be source of truth, so I don't have to hunt for it and so that I know that I'm getting the latest guidance.

As Platform Governance Team, I want make sure researchers have access to important information on Platform, since it is source of truth for research guidance.

- Intended Audience: VFS teams
- Desired Outcome: guidance moved to Platform and marked deprecated in GH
- Due Date:

Assignee: @claytonzook 
Peer reviewer: @seanasewell 

## Description
There is a [VSA GH document for research top line summary](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/design/topline-summary-template.md). We've moved that that document to [Platform Github](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/sharing-research/topline-summary-template.md) but unsure if and where it's linked on Platform website.

## Artifacts Potentially / Confirmed Needing Updates
[Research Checklist section 5 > After Sessions](https://depo-platform-documentation.scrollhelp.site/research-design/research-checklist#ResearchChecklist-5.Conductsessions)
as updated on the [Confluence Research Checklist](https://vfs.atlassian.net/wiki/spaces/RD/pages/1958773011/Research+Checklist#After-research-sessions)

## Tasks
- [x] Content audit to figure out if the old GH document is linked anywhere on Platform
- [x] If needs new links, decide where on Platform this should be linked from

## Acceptance Criteria
- [x] Platform links to the old doc is replaces with the new doc
- [x] Updates have been Peer Reviewed by Governance Team member
- [x] User Story need has been fulfilled

---

## Related Governance Team Resources
_For additional information and guidance to complete tasks, please see links below._
[Documentation Update Guidance for GitHub templates](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2047868956/Guidance+for+updating+GitHub+Issue+templates) <br />
[Documentation Update Guidance for Platform Website pages](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2089091115/Guidance+for+updating+Platform+Website+pages) <br />
[Documentation Update Guidance for Confluence pages](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2089222173/Guidance+for+updating+Confluence+pages) <br />
[Guidelines for Governance Team Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1127219219/Guidelines+for+Team+spaces) <br />
[Guidelines for Platform Products and Services Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1321140361/Guidelines+for+Platform+Products+and+Services+space) <br />
[Description of Platform Website spaces and content](https://vfs.atlassian.net/wiki/spaces/AP/pages/1241579796/Organization) <br />

---
## How to configure this issue
- [ ] **Ensure that story is attached to appropriate Initiative (if not tech debt)** (what body of work is this a part of?)
- [ ] Ensure that any additional labels are added ('tech-debt', 'collaboration-cycle')
- [ ] Ensure that story is in the correct pipeline (Backlog, Planned, In Progress, Review/QA, Closed)
- [ ] Ensure that assignees are correct so you receive all updates/comments on the ticket

**Comment by shiragoodman** [comment]
hey @claytonzook  - @CherylEvans had too many tickets on her plate for this sprint, so I'm reassigning this ticket to you. @seanasewell is still your peer reviewer. Please let me know if you have any questions!

**Comment by claytonzook** [comment]
@seanasewell ready for peer review.

I didn't find any links to the old version. I added one link to the new version of the GH doc on the Research Checklist, section 5, subsection "After Sessions"

Additionally I made the subsections in section 5 lower level headings (vs bold titles as they were previous). Since they each have so much information, I thought this would help on the accessibility front and probably otherwise generally makes sense to do.

**Comment by claytonzook** [comment]
@seanasewell ready for peer review.

I didn't find any links to the old version. I added one link to the new version of the GH doc on the Research Checklist, section 5, subsection "After Sessions"

Additionally I made the subsections in section 5 lower level headings (vs bold titles as they were previous). Since they each have so much information, I thought this would help on the accessibility front and probably otherwise generally makes sense to do.
